src/styles/blocks/card.scss

```diff
@@ -0,0 +1,124 @@
+// use nprmalize scss
+body {
+  margin: 0;
+  padding: 0;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+h3 {
+  font-weight: 500;
+}
```

Comment:
Let's move this styles to intex.scss.


Or you can create a separate file for this. 
https://www.geeksforgeeks.org/difference-between-resetting-and-normalizing-in-css/